index.js

```diff
@@ -58,10 +77,10 @@ try {
     execSync(`npm version ${branch} -m "Release ${branch} version %s"`);
 
 
-  // build the package and test it
-    execSync(`yarn`);
-    execSync(`yarn build`);
-    execSync(`yarn test`);
+  // test and build
+    execSync(`npm install`);
```

Comment:
@karkipy Why change from yarn? We will be using yarn on all our projects and will depend on `yarn.lock` file.

Reply:
wanted to test a hunch, will revert it